Dockerfile

```diff
@@ -42,11 +42,15 @@ ENV PORT= \
     TZ=UTC \
     SERVER_DESCRIPTION= \
     BACKUP_ENABLED=true \
-    BACKUP_CRON_EXPRESSION="0 0 * * *"
+    BACKUP_CRON_EXPRESSION="0 0 * * *" \
+    UPDATE_ENABLED=false \
+    UPDATE_CRON_EXPRESSION="0 * * * *" \
+    WARN_MINUTES=30
```

Comment:
Since we have a variable called `UPDATE_ON_BOOT` I think these should have the prefix `AUTO_UPDATE_`

Reply:
Added the prefix `AUTO_UPDATE_`